### PR TITLE
feat: Multi account management

### PR DESCRIPTION
**Pull request by amirch1**
### PR information

**What is the current behavior?**
No multi-account management

**What is the new behavior?**
Multi account management under the administration section

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: src/applications/administration-multi-account-app/new-account/new-account.component.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  private _createAccount(): void {
+      if (!this._accountForm.valid) {
+          return;
+      }
+
+      this._isBusy = true;
+
+      const { admin, name, email, phone, template, refid } = this._accountForm.getRawValue();
+      const partner: KalturaPartner = new KalturaPartner({
+          description: 'Multi-publishers console',
```
Comment (diamond-darrell) [inline src/applications/administration-multi-account-app/new-account/new-account.component.ts]: is this a const value? Should we translate it?

Comment (amirch1) [reply src/applications/administration-multi-account-app/new-account/new-account.component.ts]: @diamond-darrell - fixed value, taken from: https://github.com/kaltura/server/blob/584ee65e777d1d335a09e9c7e445794b9129e0e0/var_console/controllers/PartnerController.php#L40

**Comment by amirch1** [comment]
@diamond-darrell - the 2 slashes occur when the base href has only "/" like when running from localhost. Should not happen on production. Do you think we should remove the last "/" retuned from browserService.getDocumentBase() ?

**Comment by diamond-darrell** [comment]
> @diamond-darrell - the 2 slashes occur when the base href has only "/" like when running from localhost. Should not happen on production. Do you think we should remove the last "/" retuned from browserService.getDocumentBase() ?

no, if it's wokrs for the prod then it's fine by me to leave it as is for the dev

**Comment by amirch1** [comment]
@diamond-darrell - PRD specified only sorting by date for columns, no resize

**Comment by diamond-darrell** [comment]
> @diamond-darrell - PRD specified only sorting by date for columns, no resize

they've probably forgotten about it, all the high-level tables have resizing feature, but okay 🙂

**Pull request merged**